atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/futures/FuturesCombiner.java

```diff
@@ -14,26 +14,20 @@
  * limitations under the License.
  */
 
-package com.palantir.atlasdb.keyvalue.cassandra.async.futures;
+package com.palantir.atlasdb.futures;
 
 import java.util.Map;
 import java.util.Optional;
+import java.util.concurrent.ExecutorService;
 
 import com.google.common.util.concurrent.ListenableFuture;
+import com.palantir.processors.AutoDelegate;
+
+@AutoDelegate
+public interface FuturesCombiner extends AutoCloseable {
 
-/**
- * Declares an interface for combining futures which represent results of queries.
- */
-public interface CqlFuturesCombiner extends AutoCloseable {
     /**
-     * Creates a new {@code ListenableFuture} whose value is a map containing the values of all its
-     * input futures, if all succeed. Input key-value pairs for which the input futures resolve to
-     * {@link Optional#empty()} are filtered out.
-     *
-     * @param inputToListenableFutureMap query input to {@link ListenableFuture} of the query result
-     * @param <T> type of query input
-     * @param <R> type of query result
-     * @return {@link ListenableFuture} of the combined map
+     * Wraps the {@link AtlasFutures#allAsMap(Map, ExecutorService)}.
      */
```

Comment:
These docs should probably be moved to `AtlasFutures#allAsMap()`

Reply:
Moved.